inbc-program/inbc/command/command.py

```diff
@@ -129,7 +129,7 @@ def invoke_update(self, args: Any) -> None:
 
         @param args: arguments passed to command-line tool.
         """
-        super()._send_manifest(args, RESTART_CHANNEL)
+        super()._send_manifest(args, RESTART_CMD_CHANNEL)
 
     def search_response(self, payload: Any) -> None:
         """Search for keywords in response message
```

Comment:
**<h2>$\color{#FFA500}Improvements: Code Maintainability$</h2>**
**Suggestion:** The search_event method in the Command class is marked as an abstract method but provides a default implementation. Similar to the search_response method, consider removing the @abstractmethod decorator if a default implementation is desired, or ensure that subclasses provide their own implementation.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
def search_event(self, payload: Any, topic: str) -> None:
    if not search_keyword(payload, [DYNAMIC_TELEMETRY, DBS_LOG, DOCKER_NAME, DOCKER_MESSAGE]):
        logger.info('Message received: %s on topic: %s', payload, topic)
```